### PR TITLE
ovirt_disk: correct description of storage_domain

### PR DESCRIPTION
**Pull request by mnecas**
##### SUMMARY

ovirt_disk: correct description of storage_domain

https://bugzilla.redhat.com/show_bug.cgi?id=1801310
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

ovirt
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Akasurde** [comment]
@mnecas Are you going to backport this change? If yes, could you please add a changelog entry? Thanks.

File: changelogs/fragments/67454-ovirt_disk-correct-description-of-storage_domain.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "ovirt_disk: correct description of storage_domain, there is no default value of the atribute"
```
Comment (Akasurde) [inline changelogs/fragments/67454-ovirt_disk-correct-description-of-storage_domain.yml]:
```suggestion
  - "ovirt_disk: correct description of storage_domain, there is no default value of the attribute"
```

**Review by Akasurde — APPROVED** [review]
LGTM

File: changelogs/fragments/67454-ovirt_disk-correct-description-of-storage_domain.yml (hunk 1 above)
Comment (acozine) [reply changelogs/fragments/67454-ovirt_disk-correct-description-of-storage_domain.yml]: @mnecas you marked this conversation as resolved but you didn't accept the suggestion - @Akasurde is correct about the spelling of `attribute` - can you accept the suggestion?

**Pull request merged**